src/shared/constants/fluxFunctions.ts

```diff
@@ -267,7 +408,98 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
       'array.from(rows: [{_time: 2020-01-01T00:00:00Z, _value: "foo"},{_time: 2020-01-02T00:00:00Z, _value: "bar"}])',
     category: 'Inputs',
     link:
-      'https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/experimental/array/from/',
+      'https://docs.influxdata.com/influxdb/cloud/reference/flux/stdlib/experimental/array/from/',
```

Comment:
The array package was promoted from experimental. You can also change the package on line 405 to `array` (won't let me make suggestions on a line that wasn't edited)

```suggestion
      'https://docs.influxdata.com/influxdb/cloud/reference/flux/stdlib/array/from/',
```